### ISSUE TITLE
Specifying an output yields a deprecation warning

### ISSUE THREAD

**Issue by NotMyFault** [lucavb/stylelint-checkstyle-reporter#828]
Hey,

I'm using the example from the readme with my paths adjusted as needed:
`stylelint src/main/scss --custom-formatter stylelint-checkstyle-reporter -o target/stylelint-warnings.xml` but that yields
```
`output` is deprecated. Use `report` or `code` instead.
```
Other tasks with `-o` don't yield such a warning, and therefore I'm unsure whether I need to update the arguments from the readme or not 😅 
Any clue?

I'm using yarn 4.0.2 with
```json
    "stylelint": "16.1.0",
    "stylelint-checkstyle-reporter": "0.2.0",
```

**Comment by lucavb** [comment]
so they changed their API with stylelint 16 (which is their right, after all it is a major release). I am passing the second argument to the xml writer which accesses a property that triggers that warning. Good news is, it still works as planned. But I will have to make a new release of the package and basically change the usage. Stay tuned.

**Comment by NotMyFault** [comment]
Thanks for the swift response! I noticed the deprecation note in our build logs and thought I might file an issue in case it stops working one day :)

**Comment by lucavb** [comment]
So I just released version 1.0.0. The usage is mainly the same, you may want to switch to piping the stylelint output, but outside of one path change it basically stayed the same.

**Comment by NotMyFault** [comment]
> So I just released version 1.0.0. The usage is mainly the same, you may want to switch to piping the stylelint output, but outside of one path change it basically stayed the same.

Thanks!
